command/core.py

```diff
@@ -46,3 +50,23 @@ def handle_team_join(self, event_data):
             logging.info(new_id + " added to database - user notified")
         except SlackAPIError:
             logging.error(new_id + " added to database - user not notified")
+
+    def get_help(self):
+        """Get help messages and return a formatted string for messaging."""
+        message = {"text": "Displaying all available commands. To read about"
+                           " a specific command, use `/rocket COMMAND help`\n",
+                   "mrkdwn": "true"}
+        attachments = []
+        for cmd in self.__commands:
+            cmd_name = self.__commands[cmd].get_name()
+            cmd_text = "*[" + cmd_name.upper() + "]*\n\n" + \
```

Comment:
Oh, one other small thing - I'm personally not a fan of the `[ ]` around the command name, since that's often used for optional arguments and stuff like that. Is it possible to remove those?

Reply:
yep, should i remove it in the /rocket code block in the picture above too?

Reply:
Hmm, I think it's more okay there - I guess I shouldn't have said "optional arguments", but more like "things where you're supposed to fill in what you want and/or optional arguments." So in `/rocket [command] help` it makes sense, but not so much in `[User]`. 

... these things are hard to describe 😂

Reply:
Oh, one other thing I just noticed - I think it would make more sense to print out the command name that you actually type, i.e. without the uppercase.

Reply:
oki i fixed it 
![image](https://user-images.githubusercontent.com/17423870/52516543-339c5f00-2be2-11e9-8774-302c0f3117fc.png)